aikau/src/main/resources/alfresco/html/Image.js

```diff
@@ -0,0 +1,421 @@
+
+      return declare([_WidgetBase, _TemplatedMixin, _PublishOrLinkMixin], {
+
+         /**
+          * @instance
+          * @typedef {object} Dimensions
+          * @property {number} width The width property
+          * @property {number} height The height property
+          */
+
+         /**
+          * An array of the CSS files to use with this widget.
+          * 
+          * @instance
+          * @type {object[]}
+          * @default [{cssFile:"./css/Image.css"}]
+          */
+         cssRequirements: [{
+            cssFile: "./css/Image.css"
+         }],
+
+         /**
+          * The HTML template to use for the widget.
+          * 
+          * @instance
+          * @override
+          * @type {string}
+          */
+         templateString: template,
+
+         /**
+          * Any optional classes to be added to the wrapped image element.
+          *
+          * @instance
+          * @type {string|string[]}
+          * @default
+          */
+         classes: null,
+
+         /**
+          * Alt text for the image.
+          *
+          * @instance
+          * @type {string}
+          * @default
+          */
+         altText: null,
+
+         /**
+          * If we can get no other sizes, then rather than having a zero-by-zero image,
+          * make it square using this value (treated as pixels).
+          *
+          * @instance
+          * @type {number}
+          * @default
+          */
+         defaultMinimumSize: 50,
+
+         /**
+          * An optional CSS height to apply to the image node (e.g. "10px" or "50%")
+          *
+          * @instance
+          * @type {string}
+          * @default
+          */
+         height: null,
+
+         /**
+          * Any optional style rules to be applied to the wrapped image element.
+          *
+          * @type {String}
+          * @default 
+          */
+         imgStyle: null,
+
+         /**
+          * When set to true, the root node (and container of the image node) will be
+          * set to be a block level element.
+          *
+          * @instance
+          * @type {boolean}
+          * @default
+          */
+         isBlockElem: false,
+
+         /**
+          * This will be set to the natural height of the displayed image.
+          *
+          * @instance
+          * @type {number}
+          * @readonly
+          * @default
+          */
+         naturalHeight: 0,
+
+         /**
+          * This will be set to the natural width of the displayed image.
+          *
+          * @instance
+          * @type {number}
+          * @readonly
+          * @default
+          */
+         naturalWidth: 0,
+
+         /**
+          * The path to the default blank gif, using the module pathing system
+          *
+          * @instance
+          * @type {string}
+          * @default
+          */
+         pathToBlankGif: "alfresco/css/images/blank.gif",
```

Comment:
Could be, but doesn't have to be. If someone provides another value to this that will fulfil its name (i.e. it's a blank gif) then there's no reason not to let them!